### PR TITLE
Mark jpg images and fonts as binary files

### PR DESCRIPTION
**Pull request by krysal**
## Problem

I was shown the following lines as changes from `git status` but I didn't really modify those files, and is preventing me from committing.

```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .github/ISSUE_TEMPLATE/design_changes.md
	modified:   frontend/src/assets/error_images/depressed_musician.jpg
	modified:   frontend/src/assets/error_images/waiting_for_a_bite.jpg
	modified:   frontend/src/assets/fonts/Inter-Bold.woff
	modified:   frontend/src/assets/fonts/Inter-Bold.woff2
	modified:   frontend/src/assets/fonts/Inter-BoldItalic.woff2
	modified:   frontend/src/assets/fonts/Inter-Italic.var.woff2
	modified:   frontend/src/assets/fonts/Inter-Italic.woff2
	modified:   frontend/src/assets/fonts/Inter-Regular.woff
	modified:   frontend/src/assets/fonts/Inter-Regular.woff2
	modified:   frontend/src/assets/fonts/Inter-Roman.var.woff2
	modified:   frontend/src/assets/fonts/Inter-SemiBold.woff2
	modified:   frontend/src/assets/fonts/Inter-SemiBoldItalic.woff
	modified:   frontend/src/assets/fonts/JetBrainsMono-Bold.otf
	modified:   frontend/src/assets/fonts/JetBrainsMono-Bold.woff2
	modified:   frontend/src/assets/fonts/JetBrainsMono-Regular.otf
	modified:   frontend/src/static/openverse-default.jpg
	modified:   frontend/test/playwright/e2e/resources/sample_image.jpg
```

## Description

This PR includes .jpg images and font files in the `.gitattributes` so we can ignore them.
 
## Checklist

- [ ] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
Approving. I suppose we will keep the mysterious design template changes

**Comment by AetherUnbound** [comment]
> I suppose we will keep the mysterious design template changes

Looks like git is deeming it as whitespace only changes 🤔 odd!

**Review by AetherUnbound — APPROVED** [review]
Thanks for fixing this!

File: .github/ISSUE_TEMPLATE/design_changes.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-
-## Description
-<!-- Describe the changes and new elements the design proposal has. Please mention all the components and styles impacted by your idea. -->
-
-## Mockups
-<!-- Add the Figma file name and link. -->
+---
+name: Design Changes
+about: Propose design changes
+labels: "🖼️ aspect: design, ✨ goal: improvement, 🚦 status: awaiting triage"
```
Comment (dhruvkb) [inline .github/ISSUE_TEMPLATE/design_changes.md:4]: Consider adding "stack: frontend"?

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**